nix/workbench/profile/prof1-variants.jq

```diff
@@ -388,7 +388,7 @@ def all_profile_variants:
     ({ extra_desc:                     "with legacy iohk-monitoring"
      , suffix:                         "iomf"
      }|
-     .node.tracing_backend           = "iohk-monitoring"
+     .node.tracing_backend            = "iohk-monitoring"
```

Comment:
I don't follow the alignment change here?

Reply:
The line was misaligned wrt. the other node config variants.